client-side/src/components/statistics/graphBar.jsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { useQuery } from '@apollo/client';
+import { BarChart } from '@mui/x-charts/BarChart';
+import { useAppSelector } from '../../redux/store.jsx';
+import Loader from '../../stories/loader/loader.jsx'
+import { getVisitedWebsitesByDate, formatDate } from './graphsUtils.js';
+import { GET_USERS } from './queries.js';
+
+export default function GraphBar({ startDate, endDate }) {

```

Comment:
add prop types and default values if not marked as isRequired

Reply:
I added